types/kintone-utility/index.d.ts

```diff
@@ -0,0 +1,112 @@
+// Type definitions for kintone-utility 0.4
```

Comment:
You need to mention that `Type definitions for non-npm package kintone-utility 0.4`